test/framework/framework.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package helpers
+// Package framework implements utils for CAPV e2e tests.
+package framework
 
 import (
```

Comment:
Super low priority and if we do it only in an follow-up I think. I'm wondering why we need copies of core CAPI utils

(we had other places in CAPV where it looks like maybe utils in core CAPI were added later, they might have even been "upstreamed" to core CAPI from CAPV and we just never adopted them in CAPV)

Reply:
Let's take this as a follow up, in this PR i just shuffled things around